From 8ea5b2f659fc4edfe230eb95c495f83db39ac41d Mon Sep 17 00:00:00 2001
From: Craig Buchek <craig@boochtek.com>
Date: Tue, 29 Jul 2008 15:59:16 +0000
Subject: [PATCH] Fix bug #99 - extra quotes in sqlite3 adapter.

Remove quote_table_name call from SQLite3Adapter#table_structure.
---
 .../connection_adapters/sqlite3_adapter.rb         |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb b/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb
index cc9c465..0827f61 100644
--- a/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb
+++ b/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb
@@ -25,7 +25,7 @@ module ActiveRecord
   module ConnectionAdapters #:nodoc:
     class SQLite3Adapter < SQLiteAdapter # :nodoc:
       def table_structure(table_name)
-        returning structure = @connection.table_info(quote_table_name(table_name)) do
+        returning structure = @connection.table_info(table_name) do
           raise(ActiveRecord::StatementInvalid, "Could not find table '#{table_name}'") if structure.empty?
         end
       end
-- 
1.4.4.4

